doc/src/base/scopedvalues.md

```diff
@@ -17,18 +17,15 @@ concurrently.
     Scoped values were introduced in Julia 1.11. In Julia 1.8+ a compatible
     implementation is available from the package ScopedValues.jl.
 
-In its simplest form you can create a [`Base.ScopedValue`](@ref) with a
-default value and then use [`Base.with`](@ref with) or [`Base.@with`](@ref) to
-enter a new dynamic scope.
+In its simplest form you can create a [`ScopedValue`](@ref Base.ScopedValues.ScopedValue)
+with a default value and then use [`with`](@ref Base.ScopedValues.with) or
+[`@with`](@ref Base.ScopedValues.@with) to enter a new dynamic scope. The new scope will
```

Comment:
This is my second or third iteration on how to use `@ref` here. I think this might be the most correct/robust way to use `@ref` here in the manual, but I don't know for sure. Hopefully it works...

(I prefer to have the visible text just be `with` rather than `Base.with`, since `Base.with` is not marked as `public` in the `Base` module. I think the preferred pattern now for using ScopedValues is `using Base.ScopedValues`, which I agree with.)